src/components/command-bar/commands/search/components/tutorials-tab-contents/tutorial-hit.tsx

```diff
@@ -41,7 +41,7 @@ const TutorialHit = ({ hit }: TutorialHitProps) => {
 	}
 
 	const badges = products?.map(
-		(productSlug: ProductSlug) => productSlugsToNames[productSlug]
+		(productSlug: LearnProductSlug) => productSlugsToNames[productSlug]
```

Comment:
Hmm yeah - i guess they're similar, but `ProductOption` seems like more "correct" i guess since it's specifically around Learn? Personally also on the fence, and think we'd be fine with either 👍